source/docs/steeltoe-management.html.markdown

```diff
@@ -535,22 +547,26 @@ To add the Thread dump actuator middleware to the ASP.NET Core pipeline, use the
 
 The Steeltoe Heap dump endpoint can be used to generate and download a mini-dump of your application. The mini-dump can then be read into Visual Studio for analysis.
 
-> Note: At this time, dumps are only possible on the Windows operating system. When integrating with the [Pivotal Apps Manager](https://docs.pivotal.io/pivotalcf/2-0/console/index.html) you will not have the ability to obtain dumps from apps running in Linux cells. Also, the heap dump filename used by the Pivotal Apps Manager will end with the extension `.hprof` instead of the expected `.dmp` extension. This may cause problems when opening the dump with Visual Studio or some other diagnostic tool. This issue will be fixed in a future release of the Apps Manager and as a workaround you can simply rename the file to use the `.dmp` extension.
+>NOTE: At this time, dumps are only possible on the Windows operating system. When integrating with the [Pivotal Apps Manager](https://docs.pivotal.io/pivotalcf/2-0/console/index.html), you do not have the ability to obtain dumps from apps running in Linux cells. Also, the heap dump filename used by the Pivotal Apps Manager ends with the `.hprof` extension instead of the usual `.dmp` extension. This may cause problems when opening the dump with Visual Studio or some other diagnostic tool. As a workaround, you can rename the file to use the `.dmp` extension.
 
 #### 1.2.8.1 Settings
 
-Below are the settings that you can apply to the Heap dump endpoint. Each setting should be prefixed with `management:endpoints:heapdump`
+The following table describes the settings that you can apply to the Heap dump endpoint:
 
 |Key|Description|Default|
 |---|---|---|
 |id|The ID of the heap dump endpoint|`heapdump`|
-|enabled|Enable or disable heap dump management endpoint|true|
-|path|Path to the heap dump endpoint when exposed using HTTP|ID|
+|enabled|Whether to enable the heap dump management endpoint|true|
+|path|The path to the heap dump endpoint when exposed over HTTP|ID|
 |sensitive|Currently not used|false|
 
+[//]: # (TODO The preceding table was clearly copied from earlier. That would be OK, except that the path is the same, which doesn't seem likely.)
```

Comment:
@Buzzardo Is this TODO still valid? the comparison seems to be between dumps and heapdumps, which do have a slightly different prefix